sdk/sdk-common/src/protocols/implementation/LendingPool.ts

```diff
@@ -0,0 +1,41 @@
+import type { Token } from '~sdk-common/common/implementation/Token'
+import { IPool } from '../interfaces/IPool'
+import { PoolType } from '../interfaces/PoolType'
+import type { Percentage } from '~sdk-common/common/implementation/Percentage'
+import { Pool } from './Pool'
+
+export interface ILendingPoolSerialized extends IPool {
+  collateralTokens: Token[]
+  debtTokens: Token[]
+  maxLTV: Percentage
+}
+
+/**
+ * @interface LendingPool
+ * @description Represents a lending pool. Provides information about the collateral
+ *              and debt tokens
+ */
+export class LendingPool extends Pool {
+  public readonly type = PoolType.Lending
+  // List of collateral tokens to be used from the lending pool
+  public readonly collateralTokens: Token[]
+  // List of debt tokens to be used from the lending pool
+  public readonly debtTokens: Token[]
+  // Max LTV for the pool
+  public readonly maxLTV: Percentage
```

Comment:
I just removed LendingPoolBaseImpl and moved everything to LendingPool. I'm not sure about the attributes of the class, so it will likely change with the work we need to do with ProtocolManager